joblib/test/test_parallel.py

```diff
@@ -289,6 +288,13 @@ def test_exception_dispatch():
                     (delayed(exception_raiser)(i) for i in range(30)),
             )
 
+def test_nested_exception_dispatch():
+    """Ensure TransportableException objects for nested joblib cases gets propagated.
```

Comment:
> More than happy to hear good arguments for using docstrings. cc @ogrisel
> @GaelVaroquaux in case they have strong opinions on this one.

I actually feel exactly like you.